app/controllers/application_controller.rb

```diff
@@ -5,4 +5,11 @@ class ApplicationController < ActionController::Base
   respond_to :html
 
   protect_from_forgery with: :exception
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end
+  end
+  helper_method :current_user
```

Comment:
👍

Reply:
Also, I would remove the `current_user` method in ApplicationHelper

Reply:
Done in sha: 0255462